DigitalOffice/Tests/ProjectServiceTests/CommandsTests/GetProjectInfoByIdCommandTests.cs

```diff
@@ -0,0 +1,82 @@
+using Moq;
+using NUnit.Framework;
+using ProjectService.Commands;
+using ProjectService.Commands.Interfaces;
+using ProjectService.Database.Entities;
+using ProjectService.Mappers;
+using ProjectService.Mappers.Interfaces;
+using ProjectService.Models;
+using ProjectService.Repositories.Interfaces;
+using ProjectServiceUnitTests.UnitTestLibrary;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ProjectServiceUnitTests.CommandsTests
+{
+    public class GetProjectInfoByIdCommandTests
+    {
+        private IGetProjectInfoByIdCommand command;
+        private Mock<IGetProjectInfoByIdRepository> repositoryMock;
+        private IMapper<DbProject, Project> mapper;
+
+        private DbProjectManagerUser dbManagersIds;
+        private DbProjectWorkerUser dbWorkersIds;
+        private DbProject project;
+
+        private Guid projectId;
+        private Guid workerId;
+        private Guid managerId;
+
+        private string Name = "Project";
+
+        [SetUp]
+        public void Setup()
+        {
+            repositoryMock = new Mock<IGetProjectInfoByIdRepository>();
+            mapper = new ProjectMapper();
+            command = new GetProjectInfoByIdCommand(repositoryMock.Object, mapper);
+
+            workerId = Guid.NewGuid();
+            managerId = Guid.NewGuid();
+            projectId = Guid.NewGuid();
+            dbManagersIds = new DbProjectManagerUser
+            {
+                ProjectId = projectId,
+                Project = project,
+                ManagerUserId = managerId
+            };
+            dbWorkersIds = new DbProjectWorkerUser
+            {
+                ProjectId = projectId,
+                Project = project,
+                WorkerUserId = workerId
+            };
+            project = new DbProject
+            {
+                Id = projectId,
+                Name = Name,
+                WorkersUsersIds = new List<DbProjectWorkerUser> { dbWorkersIds },
+                ManagersUsersIds = new List<DbProjectManagerUser> { dbManagersIds }
+            };
+        }
+
+        [Test]
+        public void ReturnsProjectInfoSuccessfully()
+        {
+            repositoryMock
+                .Setup(x => x.GetProjectInfoById(It.IsAny<Guid>()))
+                .Returns(project);
+
+            var result = command.Execute(projectId);
+            var expected = new Project
+            {
+                Name = project.Name,
+                ManagersIds = project.ManagersUsersIds?.Select(x => x.ManagerUserId),
+                WorkersIds = project.WorkersUsersIds?.Select(x => x.WorkerUserId)
+            };
+
+            SerializerAssert.AreEqual(result, expected);
+        }
```

Comment:
one test?

Reply:
Well, I don't have any checks on this step, so there's nothing else to test, I suppose. All checks are in Repository and Mapper and there're, for example, checks if there's an exception in case if the request is null or there's no such project.

Reply:
May be you can still do this `repositoryMock.Setup(x => x.GetProjectInfoById(It.IsAny<Guid>())).Throws(new Exception());;` Check repositori that it return exception and method of command will work fine.

Reply:
And you can still mock mapper, cause the same error

Reply:
fixed